### PR TITLE
[WIP] Create AbstractViews class

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Based on #103 

Refactor all views as an abstract class.

Done:
* index
* grid

Remaining todo:
* create
* view
* edit
* delete

Concerning the views `form` and confirmation, this is not part of basic CRUD and could be done later, as an extension. This is not needed for GMF admin.

### CODE REVIEW

File: c2cgeoform/.project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (arnaud-morvan) [inline c2cgeoform/.project]: You should not commit eclipse project file and settings. I ignore them using https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

File: c2cgeoform/.pydevproject [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (arnaud-morvan) [inline c2cgeoform/.pydevproject]: Same thing, this is an eclipse file

File: c2cgeoform/.settings/org.eclipse.core.resources.prefs [hunk 3]
```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```
Comment (arnaud-morvan) [inline c2cgeoform/.settings/org.eclipse.core.resources.prefs]: Same thing, this is an eclipse file

File: c2cgeoform/templates/site/index.pt [hunk 4]
```diff
@@ -17,7 +17,8 @@
       <script type="text/javascript">
         $('#grid').bootgrid({
             ajax: true,
-            url: '${request.route_url('grid', schema=schema.name)}',
+            multiSort: true,
```
Comment (arnaud-morvan) [inline c2cgeoform/templates/site/index.pt]: here you activate multisort on client side, but this seems not supported on server side for the moment

**Comment by cmangeat** [comment]
"make build" should be preferred to "make install", or not.

**Comment by arnaud-morvan** [comment]
It seems that sbrunner prefer build, but this is not constant in all c2c projects, here will may use build to be consistent with gmf_admin.

File: c2cgeoform/templates/site/index.pt (hunk 4 above)
Comment (arnaud-morvan) [reply c2cgeoform/templates/site/index.pt]: Multisort done on server side in a2bcdf5

**Pull request merged**